src/ds5/ds5-device.cpp

```diff
@@ -895,54 +895,57 @@ namespace librealsense
                 gain_option,
                 enable_auto_exposure));
 
-        // Alternating laser pattern is applicable for global shutter/active SKUs
-        auto mask = d400_caps::CAP_GLOBAL_SHUTTER | d400_caps::CAP_ACTIVE_PROJECTOR;
-        // Alternating laser pattern should be set and query in a different way according to the firmware version
-        bool is_fw_version_using_id = (_fw_version >= firmware_version("5.12.8.100")); 
-        if ((_fw_version >= firmware_version("5.11.3.0")) && ((_device_capabilities & mask) == mask))
+        if (_pid != ds::RS405_PID)  // D405 does not have emitter
```

Comment:
Use dynamic discovery (line 901) and act according to CAP_ACTIVE_PROJECTOR.
That capability must be off for 0x0B5B